apps/css-workshop/src/components/Button.astro

```diff
@@ -35,6 +38,9 @@ const {
   data-iui-size={size}
   data-iui-variant={idea ? 'high-visibility' : variant}
   data-iui-active={isActive ? 'true' : undefined}
+  data-iui-loading={loading ? 'true' : undefined}
+  tabindex='0'
```

Comment:
nit: Just curious, was this needed for something?

Reply:
This makes it consistent with React and allows focusing disabled buttons. See https://github.com/iTwin/iTwinUI/pull/1844 and https://github.com/iTwin/iTwinUI/pull/1461